bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -11,7 +11,13 @@ export default class Whiteboard extends React.Component {
   }
 
   renderWhiteboard() {
+    let slideObj = this.props.current_slide;
```

Comment:
We should stick to just one naming convention in our case it should be camelCase